### DISCUSSION TITLE
[视频] 学科发展、学术界趋势、建议（共勉）

### DISCUSSION THREAD

**Discussion by will-ww** [kaiyuanshe/ONES#46] [资料分享]
学科发展的方式：
- 知识增长的学科发展
- 人为因素推动的学科发展

中国国际关系学界的变化趋势：
- 智库研究的基础研究在弱化（文章发了很多，知识创新越来越少，政策研究过多）
- 反对科学方法（反对进步）
- 国际关系的门槛越来越低（学校门槛降低、政策性内容越来越多、事实性多、喜欢跨界谈）
- 不用学术标准判断学术成果（用社会评判标准，因此发散思维受欢迎）

四个建议：
- 不是政策制定者，多研究原理，少研究对策
- 多学习不会的，少反对不懂的
- 多最求专业的，少最求时评的
- 多信奉学术标准，少用社会标准衡量

为了学术生命力强，找到学术乐趣，多做基础学术研究，少做政策！

以上共勉！

视频地址：[链接](https://www.bilibili.com/video/BV1n34y1879d?p=1&share_medium=android&share_plat=android&share_session_id=931f31b3-07f7-49ca-a9a8-5670fdeef07f&share_source=WEIXIN&share_tag=s_i&timestamp=1647405450&unique_k=wIiahzK&share_times=2)

**Comment by wangleo61** [comment]
这个好，**多学习不会的，少反对不懂的**
